components/script/dom/performanceresourcetiming.rs

```diff
@@ -112,24 +112,24 @@ impl PerformanceResourceTiming {
             entry: PerformanceEntry::new_inherited(
                 DOMString::from(url.into_string()),
                 DOMString::from("resource"),
-                resource_timing.start_time as f64,
-                resource_timing.response_end as f64 - resource_timing.start_time as f64,
+                resource_timing.start_time.as_nanos() as f64,
+                (resource_timing.response_end - resource_timing.start_time).as_nanos() as f64,
```

Comment:
It seems we may have been covering up a bug here where `response_end` was either 0 or less than `start_time`. Perhaps it isn't being set? Perhaps this change can either address the bug or do a `checked_duration_since` here?

Reply:
If this issue is too time-consuming, another option is to give up on storing `Instant` and `SystemTime` now and just store nanoseconds. Apologies, I didn't realize there was this annoying bug here.

Reply:
Yeah that also can be an option, I am giving it a try

Reply:
@mrobinson I used nanoseconds but I will investigate this bug further in another PR